src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html

```diff
@@ -361,45 +372,168 @@ <h3>Backup Source Configuration</h3>
                 </div>
             </div>
         </div>
-        
-        <div class="form-group row" data-bind="validationElement: backupDirectory, visible: source() !== 'cloud'">
-            <div class="col-md-4">
-                <label class="control-label">Backup Directory</label>
+
+        <!-- Azure -->
+
+        <div data-bind="visible: source() === azureCredentials().backupStorageType">
+            <div class="form-group row">
+                <div class="col-md-4">
+                    <label class="control-label">Account Name<i class="required"></i></label>
+                </div>
+                <div class="col-md-8">
+                    <input class="form-control" data-bind="textInput: azureCredentials().accountName" />
+                </div>
             </div>
-            <div class="col-md-8">
-                <input type="text" class="form-control" id="backupFolderPath"
-                       data-bind="textInput: backupDirectory, hasFocus: isFocusOnBackupDirectory, disable: $parent.databaseModel.spinners.fetchingRestorePoints()"
-                       placeholder="Enter backup directory path" />
-                
-                <ul class="dropdown-menu max-height" data-bind="foreach: $parent.backupDirectoryPathOptions, autoComplete: '#backupFolderPath'">
-                    <li data-bind="click: _.partial($root.databaseModel.backupPathHasChanged, $data)">
-                        <a href="#" data-bind="text: $data"></a>
-                    </li>
-                </ul>
+            <div class="form-group row">
```

Comment:
```with: azureCredentials```  and then you can remove azureCredentials().[PROPERTY-NAME]

Reply:
done